submissions/peter-rashid/scripts/article.js

```diff
@@ -0,0 +1,56 @@
+var articles = [];
+
+function Article (opts) {
+  // TODO: Use the js object passed in to complete this contructor function:
+  // Save ALL the properties of `opts` into `this`.
+  this.author = opts.author;
+  this.title = opts.title;
+  this.category = opts.category;
+  this.authorUrl = opts.authorUrl;
+  this.publishedOn = opts.publishedOn;
+  this.body = opts.body;
+}
+
+Article.prototype.toHtml = function() {
+  var $newArticle = $('article.template').clone();
+
+  $newArticle.attr('data-category', this.category);
+  $newArticle.attr('href', this.authorUrl);
```

Comment:
This is close! On line 18 we'll need to find the anchor first before being able to set the href attribute properly. At the moment it is defaulting to the home page.